examples/seismic/tutorials/13_making_synthetics.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to creating seismic synthetics\n",
+    "The synthetics which we will be building in this tutorial will be made with the use of GemPy, an open-source 3D geological modelling package for python. As this is not a core dependency of Devito, we will need to install it. Furthermore, you may encounter issues installing GemPy into a `conda` environment using `pip`. As such, we recommend creating a python `venv` and installing Devito in this environment using `pip` as per usual. Note that it will also be necesary to install an `ipykernel` in this environment to run this notebook. From here, we can install GemPy:"
```

Comment:
I think it's be nice to have a sentence or two right at the start with details of the contents of the notebook.

Also, I had no trouble running it in my default Devito conda env so not sure what other people are experiencing? @georgebisbas @mloubout? If that's generally the case maybe this recommendation could be removed and just the note re. `venv`'s kept?

Reply:
Changed the wording slightly to be more "If you have a problem, try this"